### PR TITLE
Expui grow 11 monthly good landing amount selector

### PR DESCRIPTION
**Pull request by mlibazisi**
# Ticket: 
https://kiva.atlassian.net/browse/GROW-11

# Setting Name:
```
mg_amount_selector
```

# Setting Value (string):
```
{"name":"MonthlyGoodAmountSelector”,”enabled":true,"startTime":"2020-01-01T10:00","endTime":"2025-01-02T10:00","control":{"key":"control","name":"Control"},"distribution":{"control”:0.5,”shown”:0.5}}
```

# Control:
control
control-mobile

# Experiment:
shown
shown-mobile

### CODE REVIEW

**Comment by mlibazisi** [comment]
> Nice! There's a problem with how the amount selector looks on mobile. But I think that CSS change can be in a separate PR.

@ryan-ludwig Cool. Is it different from the image i attached to this PR? Can you share the screenshot with me? Maybe i can do something quick. Thanks!

**Comment by ryan-ludwig** [comment]
> > Nice! There's a problem with how the amount selector looks on mobile. But I think that CSS change can be in a separate PR.
> 
> @ryan-ludwig Cool. Is it different from the image i attached to this PR? Can you share the screenshot with me? Maybe i can do something quick. Thanks!

Yeah, I'm not sure why they're different, but this is what I'm seeing
image

**Comment by mlibazisi** [comment]
> > > Nice! There's a problem with how the amount selector looks on mobile. But I think that CSS change can be in a separate PR.
> > 
> > 
> > @ryan-ludwig Cool. Is it different from the image i attached to this PR? Can you share the screenshot with me? Maybe i can do something quick. Thanks!
> 
> Yeah, I'm not sure why they're different, but this is what I'm seeing
> image

@ryan-ludwig oh, i dont think thats the mobile view. its slightly larger than mobile. But i can still fix that. Thanks for letting me know

File: src/pages/MonthlyGood/LandingFormExperiment.vue [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+			type: String,
+			default: 'default'
+		},
+		buttonText: {
+			type: String,
+			default: 'Contribute monthly'
+		},
+		minCustomAmount: {
+			type: Number,
+			default: number('minCustomAmount', 5)
```
Comment (mcstover) [inline src/pages/MonthlyGood/LandingFormExperiment.vue:107]: @mlibazisi you can just put the number 5 here... and 10000 below. No need to wrap it

File: src/pages/MonthlyGood/LandingFormExperiment.vue [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+			{{ buttonText }}
+		</kv-button>
+	</form>
+</template>
+
+
+<script>
+import { validationMixin } from 'vuelidate';
+import { required, minValue, maxValue } from 'vuelidate/lib/validators';
+import { number } from '@storybook/addon-knobs';
```
Comment (mcstover) [inline src/pages/MonthlyGood/LandingFormExperiment.vue:43]: this should be removed

**Pull request merged**